Metrics docs update (#377)
signalfx/signalfx-dotnet-tracing: docs/CHANGELOG.md

### General

- The release adds support for automatic runtime metrics.

### Breaking changes

### Enhancements

> - Add `SIGNALFX_RUNTIME_METRICS_ENABLED` configuration that enables runtime metrics.

Reviewer: ```suggestion
- Add `SIGNALFX_RUNTIME_METRICS_ENABLED` setting that enables runtime metrics.
```

In order to use the same wording as in `advanced-config.md`.

- Add `SIGNALFX_REALM` configuration that sets realm used

> when exporting telemetry directly to Splunk Observability Cloud.

Reviewer: how about adding a note that it is a convenient option that can be used instead of setting ``SIGNALFX_METRICS_ENDPOINT_URL` and `SIGNALFX_METRICS_ENDPOINT_URL` ?

Reviewer 2: Not sure about this, @pellared. The happy path is going through the collector, right?

Reviewer: Also, the description is misleading as it actually MAKES exporting telemetry directly to Splunk Observability Cloud.

> - Add `SIGNALFX_REALM` configuration that sets realm used
> when exporting telemetry directly to Splunk Observability Cloud.

Reviewer: how about

```suggestion
- Add `SIGNALFX_REALM` setting to configure the exporters 
  to send telemetry directly to the given Splunk Observability Cloud realm.
  It is a convenient setting that can be used instead of
  `SIGNALFX_METRICS_ENDPOINT_URL` and `SIGNALFX_METRICS_ENDPOINT_URL`.
```

@theletterf @lachmatt WDYT?

Reviewer 2: ```suggestion
- Add the `SIGNALFX_REALM` setting to let the instrumentation bypass
the OpenTelemetry Collector and send telemetry directly to Splunk 
Observability Cloud ingest endpoints.
```

I'd make it very clear here that it's convenient only if one wants to bypass a local collector.

> - Add `SIGNALFX_METRICS_ENDPOINT_URL` configuration that overrides
> `SIGNALFX_REALM` configuration for the metrics ingestion endpoint if needed.

Reviewer: ```suggestion
- Add `SIGNALFX_METRICS_ENDPOINT_URL` setting that overrides
the metrics exporter's endpoint.
```
